dell/src/main/java/org/apache/iceberg/dell/EcsClientProperties.java

```diff
@@ -0,0 +1,52 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.iceberg.dell;
+
+import com.emc.object.s3.S3Client;
+
+/**
+ * Property constants of catalog
+ */
+public interface EcsClientProperties {
+
+  /**
+   * Access key id
+   */
+  String ACCESS_KEY_ID = "ecs.s3.access.key.id";
```

Comment:
just want to point out, in AWS module we decided to not exposing these properties related to credentials, because this forces users to hard-code their secrets in plain text and is not a good security practice. Also this forces users to use a long-term credential instead of a session credential that will expire and has lower security burden. That is why we had the client factory class so that by default we load credentials using the default credentials chain, and users customize the client initialization process directly if they want. I don't know enough about ECS service use pattern, maybe this is a common use pattern on your side, but just keep in mind about these potential security risks.

Reply:
1. The default impl will still serialize the secret key. All workers need the credentials.
2. Users can impl their own client if they want to retrieve credentials from other services.